src/components/tooltip/Tooltip.tsx

```diff
@@ -219,6 +219,12 @@ class Tooltip extends React.Component<TooltipProps, State> {
             theme,
         } = this.props;
 
+        const tooltipText =
+            text && typeof text === 'object' ? (text as React.ReactElement).props.defaultMessage : String(text);
+
+        const ariaLabel = (React.Children.only(children) as React.ReactElement).props['aria-label'];
```

Comment:
Rather than casting this, can you make sure `.props` is defined, so there isn't the possibility runtime error if the tooltip is wrapped around a string or other non-Element node?